doc/_data/schemas/State/Meta.json

```diff
@@ -8,7 +8,8 @@
       "properties":
       {
         "Managed": { "type": "boolean", "default": false, "description": "If true, this state does not handle transitions by itself"},
-        "StepByStep": { "type": "boolean", "description": "Affects the StepByStep transition behaviour<br/>If not set, inherits the setting from its parent FSM" },
+        "StepByStep": { "type": "boolean", "default": true, "description": "Affects the StepByStep transition behaviour<br/>If not set, inherits the setting from its parent FSM" },
```

Comment:
This is unrelated to the PR but this documentation is actually wrong:
- `StepByStep` defaults to true for the main executor and false otherwise
- `StepByStep` does not inherits the setting from its parent FSM